pom.xml

```diff
@@ -855,6 +867,7 @@
         <identity.event.handler.version>1.9.34</identity.event.handler.version>
         <identity.inbound.oauth2.version>7.0.224</identity.inbound.oauth2.version>
         <identity.inbound.saml2.version>5.11.44</identity.inbound.saml2.version>
+        <identity.x509Certificate.validation.version>1.1.19-SNAPSHOT</identity.x509Certificate.validation.version>
```

Comment:
Need to update before merging

Reply:
ack

Reply:
https://github.com/wso2/identity-api-server/pull/834/commits/a1ae531ca0e31952e1a814ff391d30da8c811c06